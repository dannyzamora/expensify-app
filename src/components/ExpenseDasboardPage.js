import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpenseSummary from './ExpensesSummary'

const ExpenseDasboardPage=()=> (
    <div>
        <ExpenseListFilters/>
        <ExpenseList/>
        <ExpenseSummary/>

        
    </div>
)
export default ExpenseDasboardPage;