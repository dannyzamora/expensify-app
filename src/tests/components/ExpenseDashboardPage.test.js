import ExpenseDasboardPage from '../../components/ExpenseDasboardPage';
import React from 'react';
import {shallow} from 'enzyme';

test("SHOULD RENDER ExpenseDasboardPage",()=>{
    const wrapper = shallow(<ExpenseDasboardPage/>);
    expect(wrapper).toMatchSnapshot();
})