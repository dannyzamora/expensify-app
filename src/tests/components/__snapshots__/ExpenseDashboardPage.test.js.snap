// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SHOULD RENDER ExpenseDasboardPage 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
  <Connect(ExpenseSummary) />
</div>
`;
